env.template

```diff
@@ -24,6 +24,13 @@ PROVIDER_BASE_URL=http://host.docker.internal:5555/api/v3/webhook
 REDIS_URL=redis://0.0.0.0:6379
 #REDIS_URL=redis://redis:6379                        #docker dev
 
+# How many seconds to delay between successive scans of the chain
```

Comment:
We should add these new env vars to `docker-compose.yaml`

Reply:
Done